OxfordBodleian/Juel-Jensen/BDLaethg34.xml

```diff
@@ -379,7 +390,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                         <history>
                             <origin>
-                                <origDate notBefore="1700" notAfter="1899" evidence="lettering"/>
+                                <origDate notBefore="1700" notAfter="1899" evidence="lettering">eighteenth–nineteenth century</origDate>
+                                Unit 2 can be dated palaeographically to the eighteenth–nineteenth century.
+                                <persName ref="PRS13624TaklaG"/> is mentioned in a secondary supplication see (<ref target="#a1"/>).
```

Comment:
"see" has jumped out of the parentheses, and I suppose the ref element should have content